lib/src/main/kotlin/com/swmansion/starknet/data/types/transactions/TransactionReceipt.kt

```diff
@@ -30,16 +30,60 @@ enum class TransactionStatus {
 
     @JsonNames("UNKNOWN", "NOT_RECEIVED")
     UNKNOWN,
+
+    @JsonNames("REVERTED")
+    REVERTED,
+}
+
+@OptIn(ExperimentalSerializationApi::class)
+@Serializable
+enum class TransactionExecutionStatus {
+    // Remove normal case jsonNames when pathfinder is updated
+
+    @JsonNames("SUCCEEDED", "Succeeded")
+    SUCCEEDED,
+
+    @JsonNames("REVERTED", "Reverted")
+    REVERTED,
+
+    @JsonNames("REJECTED", "Rejected")
+    REJECTED,
```

Comment:
https://community.starknet.io/t/efficient-utilization-of-sequencer-capacity-in-starknet-v0-12-1/95607
Apparently this is only supported by gateway, since it's not in spec.